js/main.js

```diff
@@ -47,4 +49,21 @@ const fileit = (error) => {
             output.innerHTML = "Unknown Error please try again";
             break;
     }
-}
\ No newline at end of file
+}
+
+const changeBodyBg = () => {
+    if(!flag)
+    {document.body.style.background = 'black';
+    document.getElementById("title").style.color = 'white';
+    document.getElementById("myButton1").value="Light";
+    document.getElementById("myButton1").style.background = 'white';
```

Comment:
Can you intend this?

Reply:
I modified the code, to minimize size of code .Kindly have a look